aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -438,7 +449,13 @@ std::tuple<Tensor, Tensor, Tensor, Tensor> _embedding_bag_cpu_impl(
     TORCH_CHECK(per_sample_weights.numel() == indices.numel());
   }
 
-  auto bag_size = make_bag_size(offsets, indices, mode, requires_grad);
+
+  at::Tensor bag_size;
+  if (include_last_offset) {
+    bag_size = make_bag_size(offsets.slice(0, 0, offsets.size(0) - 1, 1), indices, mode, requires_grad);
```

Comment:
this make_bag_size could be optimized to do less temporary tensors, but I guess it's ok for now (maybe add a TODO/comment?)